crates/cli/src/commands.rs

```diff
@@ -105,6 +105,10 @@ pub struct BuildCommandOpts {
     /// JavaScript runtime options.
     /// Use `-J help` for more details.
     pub js: Vec<JsGroupValue>,
+
+    #[arg(short, long)]
+    /// Path of the plugin.
+    pub plugin: Option<PathBuf>,
```

Comment:
Any reason why the plugin flag is not scoped? [In the _What are the changes?_ section of the RFC](https://github.com/bytecodealliance/javy/issues/768), the plugin handling is scoped under the `-C` option group, which is what I think makes most sense, since it's easier to validate the semantic meaning of the group as a whole (e.g., `-C pluing=..` is required with `-C dynamic`)